network-api/networkapi/wagtailpages/templates/buyersguide/catalog.html

```diff
@@ -52,12 +54,39 @@ <h1 class="h1-heading text-center mt-0">{{ page.header }}</h1>
             <span class="current-creepiness"></span>
           </span>
 
-          <span id="product-filter-pni">
-            <input type="checkbox" id="product-filter-pni-toggle" autocomplete="off">
-            <label for="product-filter-pni-toggle" class="pni-checkbox-facade"></label>
-            <span class="pni-icon">&nbsp;</span>
-            <label for="product-filter-pni-toggle">{% trans "*privacy not included" %} </label>
-          </span>
+          
+        </div>
+      </div>
+      <div class="col-12 tw-mb-4 tw-px-0 tw-flex tw-items-end">
+        <span class="tw-capitalize tw-text-5xl tw-font-zilla category-header tw-mb-2">{% if current_category.parent %}{{current_category.parent.localized.name}}{% elif current_category %}{{current_category.localized.name}}{% else  %}{% trans "All" %}{% endif %}</span>
+
+        <span id="product-filter-pni">
+          <input type="checkbox" id="product-filter-pni-toggle" autocomplete="off">
+          <label for="product-filter-pni-toggle" class="pni-checkbox-facade"></label>
+          <span class="pni-icon">&nbsp;</span>
+          <label for="product-filter-pni-toggle">{% trans "*privacy not included" %} </label>
+        </span>
+      </div>
+      
+    </div>
+    <div class="row px-0 tw-mb-4">
+      <div class="tw-w-full">
+        <div class="tw-flex tw-space-x-2">
+          {% for cat in categories %}
+            {% with original=cat.original %}
+            {% if cat.parent != None %}
+              {% if original.published_product_page_count > 0 %}
+                {% localizedroutablepageurl home_page 'category-view' original.slug as cat_url %}
+                <a class="{% if current_category.name != cat.parent.name and current_category.parent.name != cat.parent.name %} tw-hidden {% endif %} subcategories tw-no-underline tw-text-gray-60 border tw-border-gray-20 tw-bg-white tw-px-2 tw-py-1 tw-font-sans tw-rounded-3xl tw-font-normal tw-text-[12px] tw-leading-[1.3] hover:tw-bg-pni-lilac hover:tw-border-pni-lilac "
```

Comment:
let's capture this giant tailwind string as a var =)
```
            {% with original=cat.original tailwind_css="tw-no-underline tw-text-gray-60 border tw-border-gray-20 tw-bg-white tw-px-2 tw-py-1 tw-font-sans tw-rounded-3xl tw-font-normal tw-text-[12px] tw-leading-[1.3] hover:tw-bg-pni-lilac hover:tw-border-pni-lilac" %}
            ...
                <a class="{% if current_category.name != cat.parent.name and current_category.parent.name != cat.parent.name %} tw-hidden {% endif %} subcategories {{ tailwind_css }}"
```

Reply:
We should also be able to determine whether to "preselect" the subcategory here: if `current_category.localized.name == cat.localized.name` then we can use the "selected" tailwind classes instead.